crates/backend/src/main.rs

```diff
@@ -24,11 +25,17 @@ use actix_web::{
 async fn main() -> std::io::Result<()> {
     let serve_from = "../../packages/playground/dist";
     if !Path::new(serve_from).is_dir() {
-        panic!("{} is not a valid directory", serve_from);
+        panic!("{} is not a valid directory \n Hint: try to run yarn build inside the Frontend Repo first.", serve_from);
```

Comment:
I really think we should keep services as independent from the environment as possible. This error message introduces knowledge of a specific environment. I think we can stay with the existing one and handle the DX at another level.

Reply:
Done